travispy/entities/_entity.py

```diff
@@ -181,6 +181,10 @@ def _load(cls, infos, session):
         for info in infos:
             entity = cls(session)
             for key, value in info.items():
+                # Log.body from Travis is empty, and is fetched on demand.
+                if key == 'body' and info['type'] == 'Log':
+                    assert value == ''
```

Comment:
This assert has highlighted that some times the Log api result does include a body.